ogr_fdw.c

```diff
@@ -583,11 +583,27 @@ ogrGetForeignPlan(PlannerInfo *root,
 	ogrFinishConnection(&(planstate->ogr));
 	
 	/* Create the ForeignScan node */
+	
+/*
+ * Require PostgreSQL >= 9.5
+ */
+#if PG_VERSION_NUM >= 90500
```

Comment:
with clever formatting you can narrow this #ifdef to being just around the new 9.5 parameter